Gruntfile.coffee

```diff
@@ -936,10 +940,29 @@ module.exports = (grunt) ->
 			all:
 				options:
 					ignore: [
-						"The “details” element is not supported properly by browsers yet. It would probably be better to wait for implementations."
 						"The “date” input type is not supported in all browsers. Please be sure to test, and consider using a polyfill."
 						"The “time” input type is not supported in all browsers. Please be sure to test, and consider using a polyfill."
 						"The “longdesc” attribute on the “img” element is obsolete. Use a regular “a” element to link to the description."
+						# TODO: Confussed language detection
+						"This document appears to be written in French but the “html” start tag has “lang=\"en\"”. Consider using “lang=\"fr\"” (or variant) instead."
+						"This document appears to be written in Czech but the “html” start tag has “lang=\"sk\"”. Consider using “lang=\"cs\"” (or variant) instead."
+						"This document appears to be written in Macedonian but the “html” start tag has “lang=\"bg\"”. Consider using “lang=\"mk\"” (or variant) instead."
+						"This document appears to be written in English but the “html” start tag has “lang=\"zh-Hans\"”. Consider using “lang=\"en\"” (or variant) instead."
+						"This document appears to be written in English but the “html” start tag has “lang=\"zh\"”. Consider using “lang=\"en\"” (or variant) instead."
+						"This document appears to be written in English but the “html” start tag has “lang=\"af\"”. Consider using “lang=\"en\"” (or variant) instead."
+						"This document appears to be written in English but the “html” start tag has “lang=\"sq\"”. Consider using “lang=\"en\"” (or variant) instead."
+						"This document appears to be written in Macedonian but the “html” start tag has “lang=\"bg\"”. Consider using “lang=\"mk\"” (or variant) instead."
+						"This document appears to be written in Nepali but the “html” start tag has “lang=\"hi\"”. Consider using “lang=\"ne\"” (or variant) instead."
+						"This document appears to be written in Macedonian but the “html” start tag has “lang=\"bg\"”. Consider using “lang=\"mk\"” (or variant) instead."
+						"This document appears to be written in Nepali but the “html” start tag has “lang=\"hi\"”. Consider using “lang=\"ne\"” (or variant) instead."
+						"This document appears to be written in Faroese but the “html” start tag has “lang=\"is\"”. Consider using “lang=\"fo\"” (or variant) instead."
+						"This document appears to be written in English but the “html” start tag has “lang=\"is\"”. Consider using “lang=\"en\"” (or variant) instead."
+						"This document appears to be written in Serbian but the “html” start tag has “lang=\"uk\"”. Consider using “lang=\"sr\"” (or variant) instead."
```

Comment:
Gonna have to ignore Dutch too to prevent Travis-CI from failing:
```
"This document appears to be written in Dutch but the “html” start tag has “lang=\"en\"”. Consider using “lang=\"nl\"” (or variant) instead."
```

Or maybe just hold off until jzaefferer/grunt-html#119 is merged.